docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,59 @@
+- use the function get_SHW_types_and_spaces to get a list of spaces for each SHW type: `shw_and_spaces_dict = get_SHW_types_and_spaces(U_RMD)`
+- look at each SHW space type: `for shw_space_type in shw_and_spaces_dict:`
+  - look at each space: `for space_id in shw_and_spaces_dict[shw_space_type]:`
+    - get the space using get_component_by_id: `space = get_component_by_id(U_RMD, space_id)`
+    - look for the ServiceWaterHeatingUse in the space.  If even one space has a ServiceWaterHeatingUse, continue to rule logic: `if len(u.space.service_water_heating_uses) > 0: CONTINUE TO RULE LOGIC`
+  - if the program reaches this line without going to the rule logic, the project is not applicable for this SHW heating use: `NOT_APPLICABLE`
+
+    ## Rule Logic: 
+    - create a boolean to keep track of whether everything matches: `all_match = TRUE`
+  - create an error string: `error_str = ""`
+  - create the compare context string: `compare_context_str = "AppG 11-1 P_RMD Equals U_RMD"`
+  - get a dictionary of all SHW equipment connected to the SHW space type in the proposed model: `p_shw_equipment_dict = get_SHW_equipment_connected_to_use_type(P_RMD)[shw_space_type]`
+  - get a dictionary of all SHW equipment connected to the SHW space type in the user model: `u_shw_equipment_dict = get_SHW_equipment_connected_to_use_type(U_RMD)[shw_space_type]`
+  - create a list of all of the SHW equipment types that exist in both the proposed and user models: `shw_equipment_types = set(p_shw_equipment_dict.keys()) | set(u_shw_equipment_dict.keys())`
+  - loop through each of the equipment types: `for shw_equipment_type in shw_equipment_types:`
+    - continue if all_match is still true: `if all_match:`
+      - check if this equipment type matches between the proposed and user models.  First check whether there are the same number equipment: `if len(p_shw_equipment_dict[shw_equipment_type]) == len(u_shw_equipment_dict[shw_equipment_type]):`
+          - look at each equipment type system in the proposed model and see if there is one that is the same in the user model.  This check relies on the understanding that the RCT team has a method for comparing all elements within an object match (compare_context_pair): `for p_SHW_equip_id in p_shw_equipment_dict[shw_equipment_type]:`
+              - if this system is in U_RMD.service_water_heating_distribution_systems, compare the systems: `if p_SHW_equip_id in u_shw_equipment_dict[shw_equipment_type]:`
+                  - use compare_context_pair to compare systems and set all_match to FALSE if the systems don't compare: `if !compare_context_pair(p_SHW_equip_id, p_SHW_equip_id, $, extra_schema_for_SHW_comparison.json, true, compare_context_str, error_str): all_match = FALSE`
+              - otherwise, set all_match to false: `else: all_match = FALSE`
```

Comment:
I think all_match = TRUE here? It is currently being set to FALSE in all conditions.